app/components/Header/index.js

```diff
@@ -1,46 +1,21 @@
-/**
- *
- * Header
- *
- */
-
 import React from 'react';
 import styled from 'styled-components';
-import {} from 'react-intl';
 import { Link } from 'react-router-dom';
+import NavModal from 'components/NavModal';
 
-// TODO this is very hacky. beware
-const StyledNavButton = styled.button `
-  border-color: rgba(0,0,0,0) !important; 
+const StyledNavButton = styled.button`
+  &&& {
```

Comment:
I don't think you these `&&& { ... }` blocks, can just replace with the inner content throughout. Can you explain why it's needed?

Reply:
I replaced the CSS here because I found a recommended technique in the documentation to avoid using "!important" (and you previously had a comment above this saying it was "hacky," so I thought I'd address it) (reference: https://www.styled-components.com/docs/faqs#how-can-i-override-styles-with-higher-specificity).

The other places where &&& is used is to increase specificity in order to override the Bootstrap styles on specific classes, and it's the only way to do that using styled-components without adding to injectglobal.